### DISCUSSION TITLE
Where is the Security Onion Ubuntu ISO?

### DISCUSSION THREAD

**Discussion by nlguerra2003** [Security-Onion-Solutions/securityonion#4345] [Unsupported Versions] [answered]
Okay, the packages cannot be installed into Ubuntu 20.04 but, when I want to use the ISO with all the information of Security Onion, the only ISO I see installs CentOS.
This ISO is the only one in VERIFY_ISO.md, and it is the target of many links (Security Onion's web page, and the documentation link to it).
Where is the Ubuntu ISO?

**Comment by presianbg** [comment, marked as answer]
The SO ISO image is based on CentOS 7, there is no other "flavor".
Not sure if 20.04 is supported:
https://github.com/Security-Onion-Solutions/securityonion/discussions/2669

**Reply by nlguerra2003** [reply]
Oh, my bad. It seems like I got confused by all the mentions to CentOS next to Ubuntu and thought _if they have CentOS, they must have Ubuntu as well_ since they tell me about both in the same sentence.

Thank you!
